3.3/.sti/bin/assemble

```diff
@@ -1,23 +1,66 @@
 #!/bin/bash
 
-# For SCL enablement
-source .bashrc
+# This script uses these environment variables:
+#
+# DISABLE_COLLECTSTATIC: if not empty, inhibits execution of 'manage.py collectstatic'.
 
-set -e
+function assemble() {
+  # For SCL enablement
+  source .bashrc
```

Comment:
@soltysh readable? so having an 'assemble' script with 'assemble()' function is readable? I think the script itself is named assemble, why we should have function inside? Why it will be pain to maintain? Having easy to read and understand bash function that are doing exactly what they should do based on their name, without too much logic hidden inside is from my PoV much easier to maintain that having one big monolithic assemble() function.